test/index.spec.ts

```diff
@@ -60,22 +60,44 @@ describe('resin-semver', () => {
 			expect(semver.compare('2.0.0+rev6', 'Resin OS 2.0.0+rev3')).to.equal(1);
 			expect(semver.compare('2.0.0+rev2', 'Resin OS 2.0.0+rev5')).to.equal(-1);
 			expect(semver.compare('2.0.0+rev3', '2.0.0+rev3')).to.equal(0);
+			expect(semver.compare('2.0.0+rev1', '2.0.0+rev11')).to.equal(-1);
+			expect(semver.compare('2.0.0+rev2', '2.0.0+rev11')).to.equal(-1);
 		});
 
 		it('should correctly compare ".dev" versions', () => {
-			expect(semver.compare('2.0.0', '2.0.0+dev')).to.equal(1);
+			expect(semver.compare('2.0.0', '2.0.0+dev')).to.equal(-1);
```

Comment:
Here we're saying that `+dev` > nothing - I don't think that should be true, assuming we're treating dev < everything else.

Reply:
I'm guessing this comes from the previous discussion  that "more specific" is higher ranked, so `+dev` is indeed > nothing. So I guess this needs some policy decision to clarify? :/